src/WebJobs.Script/Description/Binding/BindingMetadata.cs

```diff
@@ -44,6 +44,8 @@ public class BindingMetadata
         [JsonConverter(typeof(StringEnumConverter))]
         public Cardinality? Cardinality { get; set; }
 
+        public bool ReturnsSomething { get; private set; }
```

Comment:
IS this just for Node or for C# too?

Reply:
Not C#. The intent was Node, but it could also work for other non-C# languages.